snuba/datasets/storages/transactions.py

```diff
@@ -67,6 +67,7 @@
         ("contexts", Nested([("key", String()), ("value", String())])),
         ("_contexts_flattened", String()),
         ("measurements", Nested([("key", String()), ("value", Float(64))]),),
+        ("span_op_breakdowns", Nested([("key", String()), ("value", Float(64))]),),
```

Comment:
Does this need to be a Float? Is it possible for it to be an Int?

Reply:
Since we know that the values will be timestamps. We can change this to be `UInt(64)`. I'll update the span op breakdown changes on the Relay-side to emit timestamps in nanoseconds instead of milliseconds.